tests/system/feature_store/test_feature_store.py

```diff
@@ -4476,6 +4478,122 @@ def test_ingest_with_rename_columns(self):
         assert offline_features_df.equals(inspect_result)
         assert offline_features_df.equals(expected_result)
 
+    def test_merge_different_amount_of_entities(self):
+
+        feature_name = "basic_party"
+        feature_set = fstore.FeatureSet(
+            feature_name, entities=[fstore.Entity("party_id")], engine="storey"
+        )
+        data = {
+            "party_id": ["1", "2", "3"],
+            "party_establishment": ["1970", "1980", "1990"],
+        }
+        basic_party_df = pd.DataFrame(data)
+        fstore.ingest(feature_set, basic_party_df)
+
+        feature_name = "basic_account"
+        feature_set = fstore.FeatureSet(
+            feature_name,
+            entities=[fstore.Entity("party_id"), fstore.Entity("account_id")],
```

Comment:
If I switch the order of these entities, the test fails. Is that the desired behaviour?

Reply:
@gtopper , note: I am sure, it is not correct. The order of entities must be independent, the output must be the same if 
    entities=[fstore.Entity("party_id"), fstore.Entity("account_id")],
or
    entities=[fstore.Entity("account_id"), fstore.Entity("party_id")],

Reply:
@gtopper @george0st I tried to switch them and I didn't get any failure.